code/src/index.css

```diff
@@ -5,9 +5,157 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #181818;
+  color: white;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+* {
+  margin: 0;
+  padding: 0;
+}
+/* ===== HEADER ===== */
+header {
+  margin: 20px 23px;
+  border-bottom: 2px solid #252525;
+}
+
+h1 {
+  font-size: 20px;
+  margin: 10px auto;
+}
+
+/* ===== CARDS ===== */
+.App {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+}
+
+.song-card {
+  margin: 10px;
+  width: 300px;
+  display: flex;
+  flex-direction: column;
+}
+
+.song-info {
+  margin: 10px 0;
+}
+
+.song-name {
+  margin-bottom: 5px;
+  font-size: 14px;
+  font-weight: 500;
+}
+
+.song-artist {
+  font-size: 12px;
+  color: #a8a8a8;
+}
+
+/* Display structure for when there are multiple artists */
+.song-artist:not(:first-of-type, :last-of-type)::before {
+  content: ", ";
+}
+
+.song-artist:nth-last-of-type(2)::after {
+  content: " & ";
+}
```

Comment:
Really smart of you to do this part in css instead of js! Great inspiration to us, thanks!